URL-API.md

```diff
@@ -1,80 +1,293 @@
+
+```
+postype=poi&pos=<MazeMap POI ID>
+postype=point&pos=<longitude>,<latitude>,<floor>
+postype=identifier&pos=<Customer defined ID>
+```
+
+```
+http://use.mazemap.com/?campusid=1&postype=point&pos=10.40213,63.41879,0
+```
+
+* `postype` defines the format expected by the `pos` parameter. In this case, a `point`.
+* `pos` defines the geographical point in a comma-separated list.
+
+#### Warning!
+
+If not used in the correct way, simulating positions can easily confuse the user. Be careful using it for other purposes than stationary info screens.
+
+
+## Specifying the placement of 'Open in new window' link
+
+```
+http://use.mazemap.com/?campusid=1&newtablink=inside
+```
+
+* `newtablink` can be either `inside`, `outside` or `none`. It's intended to be used with iframes, e.g. when embedding maps on web sites.
+
+
+## Adding type POIs
+
+Type POIs are generic types of POIs.
+
+```
+http://use.mazemap.com/?campusid=1&typepois=9,27
+```
+
+* `typepois` specifies a comma-separated list with `typepoiids`.
 
 
-4. Defining a path
---------------------
-To start mazemap with a predefined path, use the _**starttype**_ and _**start**_ parameters along with the previous _**desttype**_ and _**dest**_.
+#### Examples of typepoiids
+
+* `4` PC
+* `7` Study area
+* `9` WC
+* `27` Bus stop
+* `28` Parking
+* `114` Power outlets
+
+A full list of type POI IDs migth be available in the future.
+
+
+## Specifying a language
+
+```
+http://use.mazemap.com/?campusid=1&lang=nb
+```
+
+* `lang` defines which language to use.
+  * `en` English
+  * `nb` Norsk Bokmål
+
+
+## Specifying a search string
+
+To start MazeMap with a predefined search string:
+
+```
+http://use.mazemap.com/?campusid=1&search=Your string
+```
+
+* `search` defines the string.
+
+
+## Specifying a custom floor level
+
+To override the default floor level:
 
 ```
-http://use.mazemap.com/?campusid=1&desttype=poi&dest=35994&starttype=point&start=10.4029047,63.4186015,0
+http://use.mazemap.com/?campusid=1&zlevel=5
 ```
-*   _**starttype**_ defines the type expected for the start parameter. In the example, a geographical _point_ outside is used, as described earlier.
-*   _**start**_ defines the geographical point in 4326 comma-separated array as described earlier. Can also be a POI.
-*   _**desttype**_ In the example, a POI is used as desttype.
-*   _**dest**_ In the example the poiID for MazeMap office is used as destination.
+
+* `zlevel` defines which floor the view is set to. This might be different from building to building, i.e. `zlevel=2` might not always mean the 2nd floor.
 
 
-5. Defining a custom name for start/destination
------------------------------------------------
-When starting MazeMap with predefined start or destination, you can input a custom string for the names of these points. I.e. if you you are linking to an office with id 404, you can specify that it should display as 'John's Office'.
+## Force accessibility path state
```

Comment:
This is only implemented for NTNU as of now, so we don't want it in our docs yet